extensions/binding/org.eclipse.smarthome.binding.ntp/README.md

```diff
@@ -0,0 +1,45 @@
+# NTP Binding
+ 
+The NTP binding is used for displaying the local date and time based update from a NTP server.
+ 
+## Supported Things
+ 
+This binding supports one Thing: ntp

```

Comment:
it is actually "one thing type" - you could potentially have multiple things (NTP servers).